pkgs/development/python-modules/plexwebsocket/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, buildPythonPackage, fetchFromGitHub, aiohttp, isPy27 }:
+
+buildPythonPackage rec {
+  pname = "plexwebsocket";
+  version = "0.0.6";
+  disabled = isPy27;
+
+  src = fetchFromGitHub {
+    owner = "jjlawren";
+    repo = "python-plexwebsocket";
+    rev = "v${version}";
+    sha256 = "1sy9khxksimcmdvghg1ksk65mkiihjvhi7m7ms2kzmy7mrg3s3i7";
+  };
+
+  propagatedBuildInputs = [ aiohttp ];
```

do you mind just adding this to each package

```suggestion
  propagatedBuildInputs = [ aiohttp ];
  
  # package does not include tests
  doCheck = false;
  
  # at least guarantee the module can be imported
  pythonImportsCheck = [
    "plexwebsocket"
  ];
```

at least we will know if the module will be imported